tests/errors/strings_with_arrows.py

```diff
@@ -0,0 +1,40 @@
+from mplee_interpreter.definitions.utils import Position
+from mplee_interpreter.errors.strings_with_arrows import string_with_arrows
+
+
+def test_if_string_with_arrows_draws_correct_amount_of_arrows():

```

Comment:
```suggestion
def test_string_with_arrows_draws_correct_amount_of_arrows():
```

Ja lubie wrapować wszystkie testy tej samej metody w klaśę i wtedy w nazwie testu juz nie umieszczam nazwy testowanej funkcji

Reply:
got it.